.ci/jenkins_job_build.sh

```diff
@@ -37,6 +37,9 @@ cd "${GOPATH}/src/${vc_repo}"
 if [ "${ghprbPullId}" ] && [ "${ghprbTargetBranch}" ]
 then
 	git fetch origin "pull/${ghprbPullId}/head" && git checkout master && git reset --hard FETCH_HEAD && git rebase "origin/${ghprbTargetBranch}"
+
+	export AUTHOR_REPO_GIT_URL="${ghprbAuthorRepoGitUrl}"
```

Comment:
I presume it was a conscious decision to export these as a different name, rather than just stick with exporting their original `ghprb` names? - just curious.

Reply:
Yes, I didn't want to end up with some Jenkins plugin naming outside of the script `jenkins_job_build.sh`